src/script/standard.cpp

```diff
@@ -190,9 +191,13 @@ bool Solver(const CScript& scriptPubKey, txnouttype& typeRet, std::vector<std::v
     // So long as script passes the IsUnspendable() test and all but the first
     // byte passes the IsPushOnly() test we don't care what exactly is in the
     // script.
-    if (scriptPubKey.size() >= 1 && scriptPubKey[0] == OP_RETURN && scriptPubKey.IsPushOnly(scriptPubKey.begin()+1)) {
+    if (scriptPubKey.size() >= 1
+     && scriptPubKey[0] == OP_RETURN){
         typeRet = TX_NULL_DATA;
-        return true;
+        if(scriptPubKey.IsPushOnly(scriptPubKey.begin()+1))
+            return true;
+        else
+            return false;
```

Comment:
Why is this change needed?

Reply:
without this change scripts with multiple `OP_RETURN` were **not** marked as `TX_NULL_DATA` and were allowed as standard by the default `TX_CUSTOM` case.